tests/core/test_reader.py

```diff
@@ -1213,3 +1213,110 @@ def test_read_write_pickle(tmp_path, data_object, ext, datasets):
     match = "Only <class 'pyvista.core.dataobject.DataObject'> are supported for pickling. Got <class 'dict'> instead."
     with pytest.raises(TypeError, match=re.escape(match)):
         pv.save_pickle('filename', {})
+
+
+def test_exodus_reader():
+    # test against mug and exodus to check different valid file
+    #  extensions: .e and .exo
+
+    fname_e = examples.download_mug(load=False)
+    fname_exo = examples.download_exodus(load=False)
+
+    e_reader = pv.get_reader(fname_e)
+    exo_reader = pv.get_reader(fname_exo)
+
+    assert isinstance(e_reader, pv.core.utilities.reader.ExodusIIReader)
+
+    assert isinstance(exo_reader, pv.core.utilities.reader.ExodusIIReader)
+
+    # Focus remainder on mug because it has multiple times
+    # and block arrays
```

Comment:
Maybe split this into two tests?

```suggestion
def test_exodus_reader_ext():
    # test against mug and exodus to check different valid file
    # extensions: .e and .exo

    fname_e = examples.download_mug(load=False)
    fname_exo = examples.download_exodus(load=False)

    e_reader = pv.get_reader(fname_e)
    exo_reader = pv.get_reader(fname_exo)

    assert isinstance(e_reader, pv.core.utilities.reader.ExodusIIReader)
    assert isinstance(exo_reader, pv.core.utilities.reader.ExodusIIReader)

```suggestion
def test_exodus_reader():
    # Use mug because it has multiple times and block arrays
    e_reader = examples.download_exodus(load=False).get_reader(fname_e)
```